### PR TITLE
Remove unused HttpHeaderClientRequestId const within timeout policy used by Azure.Data.Tables.

### PR DESCRIPTION
**Pull request by ahsonkhan**
cc @gearama

### CODE REVIEW

**Review by antkmsft — APPROVED** [review]

**Review by LarryOsterman — APPROVED** [review]

**Comment by ahsonkhan** [comment]
Waiting on @gearama to approve before merging, as the library owner :)

**Review by gearama — APPROVED** [review]

**Pull request merged**
